### DISCUSSION TITLE
Shift+click a link will open the linked tiddlers continuously without closing the other tiddlers in the story river

### DISCUSSION THREAD

**Discussion by arunnbabu81** [crazko/krystal#43] [Ideas]
`Close tiddlers to the right` - When this option in the control panel is set to yes, clicking on the first link in a tiddler will open the link to the right. Any more links you click from the same tiddler will open that link to the right after replacing the previously opened tiddler on the right. I would like to over-ride this behavior if i use "shift clicking" a link instead of "just clicking" the link. That is if i shift click the links, it should keep on opening to the right without closing the already opened tiddler on the right side.

**Comment by crazko** [comment]
I'd like to explore on this feature. Seems really helpful.
